apps/bfd-server/bfd-server-launcher/src/test/java/gov/cms/bfd/server/launcher/AppConfigurationIT.java

```diff
@@ -1,92 +1,67 @@
 package gov.cms.bfd.server.launcher;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
-import java.io.BufferedReader;
+import gov.cms.bfd.sharedutils.config.ConfigException;
+import gov.cms.bfd.sharedutils.config.ConfigLoader;
 import java.io.IOException;
-import java.io.InputStreamReader;
-import java.io.ObjectInputStream;
-import java.io.ObjectOutputStream;
 import java.io.UncheckedIOException;
-import java.net.URISyntaxException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.stream.Collectors;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
 import org.junit.jupiter.api.Test;
 
 /**
- * Unit(ish) tests for {@link AppConfiguration}.
- *
- * <p>Since Java apps can't modify their environment variables at runtime, this class has a {@link
- * #main(String[])} method that the test cases will launch as an application in a separate process.
+ * Unit tests for {@link AppConfiguration}. Has to be an integration test because it depends on the
+ * existence of a war file that isn't present when unit tests are run.
  */
 public final class AppConfigurationIT {
   /**
```

Comment:
This class requires the database so it had to stay as an IT.  However it doesn't need to fork anymore.